packages/common/src/state/dotlottie-state-machine.ts

```diff
@@ -180,7 +181,18 @@ export class DotLottieStateMachine {
                   if (typeof playbackSettings.segments === 'string') {
                     this._player.goToAndPlay(playbackSettings.segments, true);
                   } else {
-                    this._player.playSegments(playbackSettings.segments, true);
+                    const [frame1, frame2] = playbackSettings.segments;
```

Comment:
Lets not manipulate the lottie-web default behaviour

If I remove the segments from the animation and set autoplay: false, the animation still autoplays, if we could fix this bug instead of having this segment hack, it'll be better.

Inside the play() method of the common code theres no check on getOptions of the autoplay value